doc/coq.rst

```diff
@@ -456,7 +456,7 @@ lang<coq-lang>` stanza present:
 
 .. code:: dune
 
-  (lang dune 3.18)
+  (lang dune 3.19)
```

Comment:
None of these documentation changes seem to be relevant. To avoid spamming the PR with irrenevant changes I suggest leaving them out.

Reply:
yea, this had to be done due to version bump. I guess these tests were updated now to use `XXX` for versioning

Resolving this after merging main